src/pages/trash/trash.jsx

```diff
@@ -1,13 +1,26 @@
 import React from "react";
-import { SideNav } from "../../components";
-import "./trash.css";
+import { SideNav, TrashNoteCard } from "../../components";
+import { useData } from "../../context/data/data-context";
+import "../archive/archive.css";
 export function Trash() {
+  const { state } = useData();
+
   return (
     <div className="notes">
       <SideNav />
-      <div className="trash-wrapper">
-        <h4>No Notes in Trash</h4>
-      </div>
+      {state.trashListArr.length === 0 ? (
+        <div className="archive-wrapper">
+          <h4>No Notes in Trash</h4>
+        </div>
+      ) : (
+        <div className="archive-wrapper">
+          <div className="archive-list-container">
+            {[...state.trashListArr].reverse().map((curr) => (
```

Comment:
no need to spread this into an array again, you can directly write `state.trashListArr.reverse().map........`